Description: Fix typos pointed out by Lintian.
 In particular, we had spelling-error-in-binary and spelling-error-in-manpage
 warnings.
Author: Doug Torrance <dtorrance@piedmont.edu>
Last-Update: 2016-01-28

--- a/wmclockmon-config/configfile.h
+++ b/wmclockmon-config/configfile.h
@@ -7,7 +7,7 @@
 
 char *configfile =
     "# AUTOGENERATED configuration file for wmmclockmon\n"
-    "# empty lines or lines begining with # are ignored\n"
+    "# empty lines or lines beginning with # are ignored\n"
     "# booleans can take the next values : 0 1 true false yes no on off\n"
     "# and are case insensitive\n"
     "\n"
--- a/doc/wmclockmon-config.1
+++ b/doc/wmclockmon-config.1
@@ -40,7 +40,7 @@
 
 .SH FILE
 .B wmclockmon\-config
-uses one default file : ~/.wmclockmonrc. Empty lines or lines begining
+uses one default file : ~/.wmclockmonrc. Empty lines or lines beginning
 with a # are ignored.
 
 
--- a/doc/wmclockmon.1
+++ b/doc/wmclockmon.1
@@ -63,7 +63,7 @@
 override those given before.
 
 Alarms can be added automatically with the included calendar (see below and
-wmclockmon-cal(1) for more informations). Moreover the today's calendar can
+wmclockmon-cal(1) for more information). Moreover the today's calendar can
 be displayed at startup or at 00:00. In that order, the \fIMessageCmd\fP
 option is used.
 
@@ -128,9 +128,9 @@
 .TP
 .B \-s,  \-\-style <name>
 style to use for display. If \-sd is given, there is no need to give an
-extention since it is automatically given (\fI.mwcs\fP). Using this option
+extension since it is automatically given (\fI.mwcs\fP). Using this option
 to a \fI.mwcs\fP file automatically sets the styles directory if not already
-given. A subsequent use of \-sd will overwrite it. Using another extention
+given. A subsequent use of \-sd will overwrite it. Using another extension
 may give erroneous results.
 .TP
 .B \-sd, \-\-style\-dir <directory>
@@ -146,7 +146,7 @@
 don't use the current locale (use the C locale instead).
 .TP
 .B \-l,  \-\-label
-use a label instead of the current date (usefull if you have multiple
+use a label instead of the current date (useful if you have multiple
 instances running different timezones).
 .TP
 .B \-sc,  \-\-show\-cal
@@ -158,7 +158,7 @@
 
 .SH FILES
 .B wmclockmon
-uses one default file : ~/.wmclockmonrc. Empty lines or lines begining
+uses one default file : ~/.wmclockmonrc. Empty lines or lines beginning
 with a # are ignored. Entries are summarized below (default value in
 parenthesis). Booleans can be either 1/0, true/false, yes/no or
 on/off, case insensitive. A sample file is given in the source package.
@@ -224,7 +224,7 @@
 letters style file and a internet time style file. Each of them contains
 several variables and their associated values. If only the main style file
 have its extension fixed to \fI.mwcs\fP, the others can have whatever name
-you want but using those given is usefull for understanding :)
+you want but using those given is useful for understanding :)
 
 .SS MAIN STYLE FILE (.mwcs)
 
